src/shogun/mathematics/linalg/LinalgBackendBase.h

```diff
@@ -228,7 +228,20 @@ class LinalgBackendBase
 	DEFINE_FOR_ALL_PTYPE(BACKEND_GENERIC_SET_CONST, SGVector)
 	DEFINE_FOR_ALL_PTYPE(BACKEND_GENERIC_SET_CONST, SGMatrix)
 	#undef BACKEND_GENERIC_SET_CONST
-	
+
+	/**
+	 * Wrapper method that sets const values to vectors or matrices.
```

Comment:
If it is really not used, kill it.
Otherwise, having a column version of it is ok, but make sure to use memset in there